addons/mail/push-to-talk-extension/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "manifest_version": 3,
+    "name": "Discuss Push-to-talk",
+    "description": "Empowers Odoo Discuss with system-wide Push-to-Talk",
+    "version": "1.0.0.0",
+    "permissions": [
+        "storage"
+    ],
+    "action": {},
+    "background": {
+        "service_worker": "background.js",
+        "type": "module"
+    },
+    "icons": {
+        "128": "assets/icons/inactive_icon.png"
+    },
+    "content_scripts": [
+        {
+            "matches": [
+                "https://*/*",
+                "http://*/*"
+            ],
+            "js": [
+                "content.js"
+            ],
+            "run_at": "document_start"
+        }
+    ],
+    "commands": {
+        "ptt-pressed": {
+            "suggested_key": "Alt+Shift+4",
+            "description": "Push-to-Talk"
+        },
+        "toggle-voice": {
+            "suggested_key": "Alt+Shift+3",
+            "description": "Voice Toggle"
```

Comment:
These keys change tab in my browser, we should try to find suggestions that do nothing in most systems by default.

Reply:
Changed for `Alt+Shift+Y/U`. Not ideal combinations, but those are just suggestions. Everyone will be able to set this according to their preferences or better yet  map the shortcut with `AutoKey` to bypass extension shortcut limitations.